app/assets/stylesheets/common/core.scss

```diff
@@ -498,6 +498,8 @@ option.uncollected {
 .flash-content {
   display: inline-block;
   padding-left: 5pt;
+  vertical-align: middle;
+  max-width: calc(100% - 3em);
```

Comment:
Keep all rules alphabetically ordered (I'm surprised the pre-commit hooks didn't pick up on this, but that's something I'll look into separately!)